docs/team/javonteo.md

```diff
@@ -1,38 +1,41 @@
 ---
-layout: page
-title: Javon Teo's Project Portfolio Page
+Javon Teo's Project Portfolio Page
```

Comment:
I believe the removed lines are necessary for our project website to render of your portfolio page correctly

Reply:
Resolved in the next commit